FAQUse: What coinjoin strategy should I select (#1305)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

Depending on many factors, like the `Anonymity score target`, the `coinjoin strategy`, and the amount of bitcoin, it can take from a couple hours to multiple days.
:::

::::details
### What coinjoin strategy should I select?

> There is no answer for all, or a "best" strategy, as is depends per user.

Reviewer: `as is depends per user` this seems not correct to me.

Author: what else

> There is no answer for all, or a "best" strategy, as is depends per user.

Reviewer: ```suggestion
There is no answer for all, or a "best" strategy, it depends on the user 's threat model.
```

Author: > it depends on the user 's threat model

not newbie friendly imo

Reviewer: I updated the suggestion, check it.

Author: not only privacy needs, also time preference

Reviewer: All the coinjoin strategy settings are privacy related. Isn't that called coinjoin time preference?

Author: I don't understand

Reviewer: You said: `not only privacy needs, also time preference` and I said: `All the coinjoin strategy settings are privacy related`, so my suggestion makes sense as it is.

Author: > All the coinjoin strategy settings are privacy related, so my suggestion makes sense as it is

no
the coinjoin time preference is for costs, not for privacy

Reviewer: Yes but it is the cost of the coinjoin.

> The coinjoin strategy should be selected according to the users prefence:

Reviewer: ```suggestion
The coinjoin strategy should be selected according to the user's preference:
```
